### PR TITLE
fix(ci): Force quotes in yaml [Take 2]

### PR DESCRIPTION
**Pull request by sindrig**
This reverts commit f72fe1d361678eeed28e0a6c6357fe7e87e72a12.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -172,7 +172,7 @@
     "isomorphic-unfetch": "3.0.0",
     "js-base64": "2.5.2",
     "js-cookie": "2.2.1",
-    "js-yaml": "3.14.0",
+    "js-yaml": "4.1.0",
```
Comment (eirikurn) [inline package.json:175]: Please upgrade `@types/js-yaml` as well.

**Review by eirikurn — CHANGES_REQUESTED** [review]
There are a some more places calling `safe*` in the codebase, eg in the API Catalogue. 

Hopefully we'll find them when we upgrade `@types/js-yaml`, but might want to look through all imports/usage to be safe (not that much).

**Review by eirikurn — APPROVED** [review]
Sweet!

**Review by pshomov — APPROVED** [review]
LGTM

**Review by saevar43 — APPROVED** [review]
LGTM

**Review by arnorhs — APPROVED** [review]

**Pull request merged**